docs/components/PageLayout.tsx

```diff
@@ -25,36 +26,33 @@ export function PageLayout({
 	const router = useRouter();
 	return (
 		<Content>
-			<Flex
-				gap={{ xs: 1, md: 2, lg: 6 }}
-				flexDirection={{ xs: 'column', md: 'row' }}
-			>
+			<Columns>
 				{sideNav && (
-					<Box
-						width={{ md: '25%' }}
-						minWidth={'10rem'}
-						maxWidth={{ md: '20rem' }}
-						flexShrink={0}
-					>
-						<SideNav
-							variant="light"
-							activePath={router.asPath}
-							title={sideNav.title}
-							titleLink={sideNav.titleLink}
-							items={sideNav.items}
-						/>
-					</Box>
+					<>
+						<Column columnSpan={{ xs: 12, md: 3 }} flexShrink={0}>
+							<SideNav
+								variant="light"
+								activePath={router.asPath}
+								title={sideNav.title}
+								titleLink={sideNav.titleLink}
+								items={sideNav.items}
+							/>
+						</Column>
+					</>
```

Comment:
The Fragment is not needed here.

```suggestion
			<Column columnSpan={{ xs: 12, md: 3 }} flexShrink={0}>
				<SideNav
					variant="light"
					activePath={router.asPath}
					title={sideNav.title}
					titleLink={sideNav.titleLink}
					items={sideNav.items}
				/>
			</Column>
```